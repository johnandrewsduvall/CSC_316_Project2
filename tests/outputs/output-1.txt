+ 10
id = 1
