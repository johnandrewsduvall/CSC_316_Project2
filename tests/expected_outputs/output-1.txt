+ 10
id = 1
+ 10
id = 2
+ 20
id = 3
+ 20
id = 4
